kani-compiler/src/codegen_cprover_gotoc/context/current_fn.rs

```diff
@@ -108,6 +104,11 @@ impl<'tcx> CurrentFnCtx<'tcx> {
     pub fn sig(&self) -> PolyFnSig<'tcx> {
         self.sig
     }
+
+    /// The body of the function.
+    pub fn body(&self) -> &Body {
+        &self.body
+    }
```

Comment:
If I'm not mistaken, this should be either `mir_stable` or `body_stable`, but for some reason you've preferred to keep the variables `mir` and `body` separate instead of adding the suffix. What's the reason?

Reply:
I think body is a better name and I didn't want to bother renaming `mir` to `mir_internal` to reduce the amount of changes here. But I can if you prefer. :)